spot-electron/src/bt-beacon/btbeacon.js

```diff
@@ -0,0 +1,61 @@
+const { config } = require('../config');
+
+const { clientController } = require('../client-control');
+const { logger } = require('../logger');
+const { joinCodeToVersion } = require('../utils');
+
+const daemon = require('./daemon');
+
+/**
+ * Represents a soft-beacon emulated by the computer running Spot TV
+ */
+class BTBeacon {
+
+    /**
+     * Instantiates a new instance of the class.
+     */
+    constructor() {
+        this.region = config.getValue('beacon.region');
+
+        if (daemon && this.region) {
+            daemon.connect().then(() => {
+                clientController.on('updateJoinCode', ({ joinCode }) => {
+                    this.joinCodeUpdated(joinCode);
+                });
+            });
+        } else {
+            logger.warn('BT beacon functionality is not available.');
```

Comment:
From my knowledge, which is years old, windows machines might not have bluetooth.  Should there be something exposed in the Spot-TV UI indicating lack of bluetooth support or that bluetooth is being broadcast? I can add it if need be but I don't know if there is a plan already to add it.

Reply:
I don't exactly know. Yeah, I guess it would be great to have it. Let's add a story for that!

Reply:
Okay, I'll make a ticket and just do it one day.